build_tools/python/e2e_test_artifacts/cmake_generator/iree_rule_generator_test.py

```diff
@@ -126,19 +126,19 @@ def test_generate_rules(self):
         entry_function="main",
         input_types=["1xf32"])
     model_b = common_definitions.Model(
-        id="5678_v1",
-        name="tf_m",
+        id="5678",
+        name="mhlo_m",
         tags=[],
-        source_type=common_definitions.ModelSourceType.EXPORTED_TF_V1,
-        source_url="https://example.com/xyz_saved_model",
+        source_type=common_definitions.ModelSourceType.EXPORTED_MHLO_MLIR,
+        source_url="https://example.com/xyz_mhlo.mlir",
         entry_function="predict",
         input_types=["2xf32"])
     model_c = common_definitions.Model(
```

Comment:
nit: This case can be removed now (we created two cases for tfv1 and tfv2).

Reply:
Removed

Reply:
Removed